rules-java-ee/addon/src/main/resources/legacy/org.jboss.windup.rules.apps.legacy.java.JBossConfig.windup.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
```

Comment:
This file should not be here. If changes are made, they should be made to the windup-rulesets repository.